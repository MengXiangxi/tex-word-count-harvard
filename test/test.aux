\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{lake_tailoring_1998}
\citation{lock_visibility_2014}
\citation{renfrew_investigations_1979}
\citation{lake_tailoring_1998}
\citation{lock_visibility_2014}
\citation{lock_visibility_2014}
\@writefile{toc}{\contentsline {section}{\numberline {1}cite\_parser}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}subjective\_cite\_parser}{1}{section.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}partially\_cite\_parser}{1}{section.3}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Word Count}{1}{section.4}}
\bibstyle{agsm}
\bibdata{references}
\harvardcite{lake_tailoring_1998}{Lake, Wosodman \harvardand \^^MMithen}{Lake et~al.}{1998}
\harvardcite{lock_visibility_2014}{Lock, Kormann \harvardand \^^MPouncett}{Lock et~al.}{2014}
\harvardcite{renfrew_investigations_1979}{Renfrew}{Renfrew}{1979}
