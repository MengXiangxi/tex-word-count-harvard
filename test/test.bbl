\begin{thebibliography}{xx}

\harvarditem[Lake et~al.]{Lake, Wosodman \harvardand\
  Mithen}{1998}{lake_tailoring_1998}
Lake, M.~W., Wosodman, P.~E. \harvardand\ Mithen, S.~J.  \harvardyearleft
  1998\harvardyearright , `Tailoring {GIS} {Software} for {Archaeological}
  {Applications}: {An} {Example} {Concerning} {Viewshed} {Analysis}', {\em
  Journal of Archaeological Science} {\bf 25}(1),~27--38.
\newline\harvardurl{\url{https://www.sciencedirect.com/science/article/pii/S0305440397901977}}

\harvarditem[Lock et~al.]{Lock, Kormann \harvardand\
  Pouncett}{2014}{lock_visibility_2014}
Lock, G., Kormann, M. \harvardand\ Pouncett, J.  \harvardyearleft
  2014\harvardyearright , Visibility and movement: towards a {GIS}-based
  integrated approach, {\em in} `Computational {Approaches} to the {Study} of
  {Movement} in {ArchaeologyTheory}, {Practice} and {Interpretation} of
  {Factors} and {Effects} of {Long} {Term} {Landscape} {Formation} and
  {Transformation}', De Gruyter, Berlin, Boston, pp.~23--42.
\newline\harvardurl{\url{https://www.degruyter.com/view/books/9783110288384/9783110288384.23/9783110288384.23.xml}}

\harvarditem{Renfrew}{1979}{renfrew_investigations_1979}
Renfrew, C.  \harvardyearleft 1979\harvardyearright , {\em Investigations in
  {Orkney}}, Reports of the {Research} {Committee} of the {Society} of
  {Antiquaries} of {London} ; no. 38, Society of Antiquaries of London, London.

\end{thebibliography}
