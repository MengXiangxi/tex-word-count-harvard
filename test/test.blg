This is BibTeX, Version 0.99d (TeX Live 2015/Debian)
Capacity: max_strings=35307, hash_size=35307, hash_prime=30011
The top-level auxiliary file: test.aux
The style file: agsm.bst
Database file #1: references.bib
You've used 3 entries,
            2909 wiz_defined-function locations,
            662 strings with 5697 characters,
and the built_in function-call counts, 11456 in all, are:
= -- 1836
> -- 164
< -- 0
+ -- 863
- -- 49
* -- 976
:= -- 2328
add.period$ -- 3
call.type$ -- 3
change.case$ -- 41
chr.to.int$ -- 3
cite$ -- 3
duplicate$ -- 56
empty$ -- 109
format.name$ -- 57
if$ -- 2062
int.to.chr$ -- 5
int.to.str$ -- 0
missing$ -- 4
newline$ -- 14
num.names$ -- 32
pop$ -- 14
preamble$ -- 1
purify$ -- 45
quote$ -- 0
skip$ -- 44
stack$ -- 0
substring$ -- 2564
swap$ -- 11
text.length$ -- 42
text.prefix$ -- 0
top$ -- 0
type$ -- 10
warning$ -- 0
while$ -- 68
width$ -- 0
write$ -- 49
