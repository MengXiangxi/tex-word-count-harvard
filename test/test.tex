\documentclass[a4paper]{article}
\usepackage{geometry} \geometry{a4paper}
\usepackage[parfill]{parskip}
\usepackage{graphicx}
\usepackage[hidelinks]{hyperref}

\usepackage[abbr]{harvard}


\title{Test File}
\author{}
\date{}

\begin{document} \maketitle

\par{\large{\textbf{First of all, all citations here are apparently wrong. Do not be serious about it. xD}}}\\

\section{cite\_parser}
\par{This is the first example \cite{lake_tailoring_1998}, and this is the second example \citeaffixed{lock_visibility_2014}{e.g.}. }

\section{subjective\_cite\_parser}
\par{\citeasnoun{renfrew_investigations_1979} wrote this book in 1979. \possessivecite{lake_tailoring_1998} article was published in 1998. }

\section{partially\_cite\_parser}
\par{This article is written by \citename{lock_visibility_2014}. We can only cite the year of publication \citeyear{lock_visibility_2014}.}

\section{Word Count}
\par{This article has 9 words in headings.}
\par{This article has no word in subsections}
\par{There are 101 words in the main part (including this part). }
\par{This article has 110 words in total.}

\newpage
\bibliographystyle{agsm}
\bibliography{references}
\end{document}