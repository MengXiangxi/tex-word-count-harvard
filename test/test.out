\BOOKMARK [1][-]{section.1}{cite\137parser}{}% 1
\BOOKMARK [1][-]{section.2}{subjective\137cite\137parser}{}% 2
\BOOKMARK [1][-]{section.3}{partially\137cite\137parser}{}% 3
\BOOKMARK [1][-]{section.4}{Word Count}{}% 4
